### PR TITLE
Tantalum change

### PR DESCRIPTION
**Pull request by Dream-Master**
change tantalum capacitor recipes

### CODE REVIEW

**Comment by Dream-Master** [comment]
feedback are welcome.

**Comment by iouter** [comment]
do you test it?

**Comment by Dream-Master** [comment]
The recipes works but i am not sure about balance

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 1]
```diff
@@ -19062,6 +19062,116 @@ public void run3() {
...
+                    },
+                    Materials.MelodicAlloy.getMolten(1152),
+                    GT_ModHandler.getModItem("EnderIO", "itemBasicCapacitor", 1L, 4),
+                    200,
+                    7680);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.VanadiumGallium, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
+                        ItemList.Gravistar.get(1L),
```
Comment (iouter) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: Neutronium is too late, it needs a uv ebf.

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java [hunk 2]
```diff
@@ -19062,6 +19062,116 @@ public void run3() {
...
+                        GT_Utility.getIntegratedCircuit(1)
+                    },
+                    Materials.MelodicAlloy.getMolten(1152),
+                    GT_ModHandler.getModItem("EnderIO", "itemBasicCapacitor", 1L, 4),
+                    200,
+                    7680);
+            GT_Values.RA.addAssemblerRecipe(
+                    new ItemStack[] {
+                        GT_OreDictUnificator.get(OrePrefixes.bolt, Materials.VanadiumGallium, 4),
+                        GT_OreDictUnificator.get(OrePrefixes.foil, Materials.Manganese, 4),
```
Comment (iouter) [inline src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19166]: Now all need manganese foil, consider some plastic foil? such as pvc, ptfe, pbi and so on.

**Comment by POPlol333** [comment]
Hm, aren't those foil are mv ? why neutronium?....

**Comment by iouter** [comment]
> Hm, aren't those foil mv ? why neutronium?....

that's stellar capactior's recipe

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 1 above)
Comment (POPlol333) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: > > Hm, aren't those foil mv ? why neutronium?....
> 
> that's stellar capactior's recipe

well so luv or zpm iirc?

**Comment by POPlol333** [comment]
> > Hm, aren't those foil mv ? why neutronium?....
> 
> that's stellar capactitor's recipe

well luv or zpm then?

**Comment by iouter** [comment]
> > > Hm, aren't those foil mv ? why neutronium?....
> > 
> > 
> > that's stellar capactitor's recipe
> 
> well luv or zpm then?

luv

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: where do you see Neutronium ?

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 2 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19166]: Sure we can add other fluids

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 1 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: The gravistar was there already if you mean this. Suggest another item

Comment (iouter) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: gravi star need Molten Neutronium, 288L

Comment (iouter) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: Quantum Star is enough, you don't seem to be using it.

Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19167]: changed

**Comment by POPlol333** [comment]
I'm personally fine with everything overall now

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 2 above)
Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19166]: But the alloys are ender io alloys and not do often used.

**Comment by Dream-Master** [comment]
> I'm personally fine with everything overall now

Then approve it please 😎

**Review by boubou19 — DISMISSED** [review]

**Comment by POPlol333** [comment]
I could have approved but the fact is you asked /core and not /developper so I didn't had button for it

**Review by POPlol333 — APPROVED** [review]
hop lol

**Review by boubou19 — APPROVED** [review]

File: src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java (hunk 2 above)
Comment (iouter) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19166]: i mean these manganese foil, not the fluid. but doesn't matter

Comment (Dream-Master) [reply src/main/java/com/dreammaster/gthandler/GT_MachineRecipeLoader.java:19166]: Ok sure I can make a new pr

**Pull request merged**